crates/nargo/tests/test_data/type_aliases/src/main.nr

```diff
@@ -0,0 +1,10 @@
+use dep::std;
+
+type foo = u8;
+
+fn main(x : Field, y : pub Field) {
+    let a: foo = 1;
+    let b: foo = 2;
+    constrain x != y;
+    constrain a != b;
```

Comment:
Lets change this test to remove `y` and change `x` to be a `u8`. Then we could use `a` and `x` together without casts, confirming that `a` is a `u8` internally. `b` and `y` would also no longer be needed then.